st3/sublime_lib/settings_dict.py

```diff
@@ -49,6 +49,15 @@ def __iter__(self) -> NoReturn:
         """Raise NotImplementedError."""
         raise NotImplementedError()
 
+    def __eq__(self, other) -> bool:
+        """Return ``True`` if `self` and `other` are of the same type
+        and refer to the same underlying settings data.
+        """
+        return (
+            type(self) == type(other) and
+            self.settings.settings_id == other.settings.settings_id
```

Comment:
Huh, do we not use line breaks before binary operators?

Reply:
Fixed.